virtualization/windowscontainers/troubleshooting.md

```diff
@@ -15,9 +15,9 @@ ms.assetid: ebd79cd3-5fdd-458d-8dc8-fc96408958b5
 Having trouble setting up your machine or running a container? We created a PowerShell script to check for common problems. Please give it a try first to see what it finds and share your results.
 
 ```PowerShell
-Invoke-WebRequest https://raw.githubusercontent.com/Microsoft/Virtualization-Documentation/master/windows-server-container-tools/Debug-ContainerHost/Debug-ContainerHost.ps1 -UseBasicParsing | Invoke-Expression
+Invoke-WebRequest http://aka.ms/Debug-ContainerHost.ps1 -UseBasicParsing | Invoke-Expression
```

Comment:
I prefer using `https://` here. I can't see a problem, the TLS cert looks good with `curl`, Chrome, and SSL check https://www.ssllabs.com/ssltest/analyze.html?d=aka.ms